cssdb.json

```diff
@@ -41,18 +41,20 @@
     "stage": 2,
     "browser_support": {
       "edge": "79",
-      "firefox": "50",
-      "chrome": "65",
-      "safari": "9",
-      "opera": "52",
-      "ios_saf": "9",
+      "firefox": "3",
+      "chrome": "15",
+      "safari": "7",
```

Comment:
7 is incorrect.

mdn has correct stats : https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link#browser_compatibility

https://mrhenry.github.io/web-tests/#75fe312b-386d-4648-b6cf-1a588a5cee3b

<img width="1032" alt="Screenshot 2022-01-20 at 11 38 53" src="https://user-images.githubusercontent.com/11521496/150322927-99ff7f70-876e-4f5e-b4c0-58b08a050476.png">